### PR TITLE
added ACC 2018 ortho imagery

### PR DESCRIPTION
**Pull request by ianvg**
Added ACC 2018 ortho imagery converted to TMS format. Need to rename the license file.

### CODE REVIEW

File: sources/north-america/us/ga/ACC_Data.geoJSON [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "max_zoom": 20,    	      
+    "attribution": {
+        "url": "https://data-athensclarke.opendata.arcgis.com/",
+        "text": "Athens-Clarke County GIS",
+        "required": true
+    },
+    "valid-georeference": true,
+    "start_date": "2019",
+    "end_date": "2021",
+    "license_url": "https://github.com/osmlab/editor-layer-index/blob/bab9f11b444f0cba6593cd50c25d0cac5aae9fa4/sources/north-america/us/ga/Athens-Clarke_County_Data.pdf",
```
Comment (andrewharvey) [inline sources/north-america/us/ga/ACC_Data.geoJSON]: I see in the email they said "are free to use for OSM projects" personally I'm not sure if that's sufficient or not.

For CC BY ideally they could complete the CC BY waiver form from https://wiki.osmfoundation.org/wiki/Licence/Waiver_and_Permission_Templates

Comment (ianvg) [reply sources/north-america/us/ga/ACC_Data.geoJSON]: Okay, thanks for the heads up. I'll see if I can add a signed waiver to here.

Comment (ianvg) [reply sources/north-america/us/ga/ACC_Data.geoJSON]: And just for future reference, is it okay if I just leave the link in the geojson as a permalink to the .pdf file?

Comment (rbuffat) [reply sources/north-america/us/ga/ACC_Data.geoJSON]: @ianvg The permalink in the pdf is ok. Even better would be to upload the pdf to the osm wiki. This allows being more independent of future GitHub changes.

Comment (ianvg) [reply sources/north-america/us/ga/ACC_Data.geoJSON]: If anyone looks at this pull request- I am still attempting to obtain a letter allowing use of the county’s satellite imagery.

**Comment by ianvg** [comment]
Woops, I accidentally clicked resolve conversation- and I’m not sure what happened to the comments for this pull request- anyways- I’m still waiting on getting permission to use this imagery.

File: sources/north-america/us/ga/ACC_Data.geoJSON (hunk 1 above)
Comment (ianvg) [reply sources/north-america/us/ga/ACC_Data.geoJSON]: [OSM_Imagery_Waiver_signed.pdf](https://github.com/osmlab/editor-layer-index/files/7247127/OSM_Imagery_Waiver_signed.pdf)

Comment (ianvg) [reply sources/north-america/us/ga/ACC_Data.geoJSON]: Dang, I’m getting intimidated by GitHub again. I’ll see if I can figure out how to add this file to my pull request.

**Review by andrewharvey — APPROVED** [review]

**Comment by cicku** [comment]
@ianvg 

Please upload the license file onto OpenStreetMap wiki, then creating a single page undernearth your State linking the file with all relevant information. The OSMLab is not part of the [openstreetmap organization](https://github.com/openstreetmap), and a large portion of OSM users are not even iD based.

The page name, for example, can be:

https://wiki.openstreetmap.org/wiki/Georgia_(U.S._state)/Athens-Clarke_County_Aerial_Imagery

Here is an example of such kind of page: https://wiki.openstreetmap.org/wiki/Pennsylvania/PEMA_Orthoimagery. Feel free to reuse it for ACC.

Finally, set license_url to the page you created.

**Pull request merged**